### ISSUE TITLE
Allow shortcuts in file manager

### ISSUE THREAD

**Issue by ghost** [jupyterlab/jupyterlab#2814]
Hello,
  In file manager, there should be shortcuts like f2 for rename, ctrl+c for copy, etc, which are not in the current file manager.

**Comment by pradyumnac** [comment]
Any update on this? This would be really handy. Not sure why no body else commented here

**Comment by dzubo** [comment]
@ian-r-rose , @pradyumnac 

Worked for me when I saved in
Settings -> Keyboard shortcuts -> User Preferences
this:
```
{
    "shortcuts": [
        {
            "command": "filebrowser:rename",
            "keys": [
                "F2"
            ],
            "selector": "body"
        }
    ]
}
```

Probably reload of the browser webpage will be needed to make it work for the first time after saving the settings.

My environment:
```
> conda list | grep lab
jupyterlab                1.0.1                    py37_0    conda-forge
jupyterlab_server         1.0.0                      py_1    conda-forge
```

**Comment by jasongrout** [comment]
Perhaps you want to make the selector a little more specific to the filebrowser, so F2 doesn't trigger a rename everywhere inside jlab? (Or maybe that's what you want?)

Perhaps `"selector": ".jp-DirListing"` or something like that would work well?

**Comment by dzubo** [comment]
@jasongrout, yes, everywhere inside jlab is good for me.

**Comment by krassowski** [comment]
This is now implemented, primarily by #10206, with follow up in #6859, #10280 and #10466. The shortcuts will be active by default and show up in the context menu (when opened over files) in the upcoming 3.1 release (now in beta).
